Jenkinsfile

```diff
@@ -93,6 +106,18 @@ pipeline {
                             orgToken: "${WHITESOURCE_ORG_TOKEN}")
                     }
                 }
+                stage('WhitesourceScan - Operator APIs') {
+                    steps {
+                        whitesourceExecuteScan(script: this,
+                            scanType: 'golang',
+                            productName: "${WSS_PROD_NAME}",
+                            userTokenCredentialsId: 'interoperator_whitesource_test_id',
+                            configFilePath: './wss-unified-agent.config',
```

Comment:
`wss-unified-agent.config` file must be made in `./operator-apis/` folder. Can be copied from `interoperator` folder